src/components/molecules/AssetListTitle.tsx

```diff
@@ -28,15 +28,15 @@ export default function AssetListTitle({
 
     const source = axios.CancelToken.source()
 
-    async function getDDO() {
-      const ddo = await retrieveDDO(did, config.metadataCacheUri, source.token)
-
-      if (!ddo) return
-      const { attributes } = ddo.findServiceByType('metadata')
-      setAssetTitle(attributes.main.name)
+    async function getAssetName() {
+      getAssetsNames([did], config.metadataCacheUri, source.token).then(
```

Comment:
just for consistency, we usually use async/await for resolving promises so something like this would also end up shorter:

```js
const title = await getAssetsNames()
setAssetTitle(title)
```